frontend/src/components/application/AdditionalInfo.tsx

```diff
@@ -213,6 +213,42 @@ const AdditionalInfo = ({
               : undefined,
           ]}
         />
+      )}
+       {data.urls?.vcs_browser && (
+        <ListBox
+          appId={appId}
+          items={[
+            data.urls.vcs_browser
+              ? {
+                icon: <MdWeb />,
+                header: t('vcs_browser'),
```

Comment:
You'll need to add this and the `contribute` one to the english translation file.